hack/aws-acceptance-test-cleanup/main.go

```diff
@@ -379,11 +377,17 @@ func realMain(ctx context.Context) error {
 		})
 		for _, igw := range igws.InternetGateways {
 			fmt.Printf("Internet gateway: Detaching from VPC... [id=%s]\n", *igw.InternetGatewayId)
-			_, err := ec2Client.DetachInternetGatewayWithContext(ctx, &ec2.DetachInternetGatewayInput{
-				InternetGatewayId: igw.InternetGatewayId,
-				VpcId:             vpcID,
-			})
-			if err != nil {
+			if err := destroyBackoff(ctx, "Internet Gateway", *igw.InternetGatewayId, func() error {
```

Comment:
Not to be address here but are there several more of these resources that need to be cleaned up with a backoff also?  How about on GKE?

Reply:
On GKE we only need to cleanup k8s clusters because we don't create a separate VPC or any vpc-related resources I think.